packages/@ama-sdk/schematics/schematics/typescript/create/index.ts

```diff
@@ -6,6 +6,7 @@ import { NgGenerateTypescriptSDKCreateSchematicsSchema } from './schema';
 /**
  * Generate a typescript SDK repository and source code base on swagger specification
  *
+ * @deprecate use @ama-sdk/create package instead
```

Comment:
I think it's this one we said we can remove. The package was not published before 9.x